### PR TITLE
fix github secret loading

### PR DESCRIPTION
**Pull request by BenTheElder**
/area prow
/hold

### CODE REVIEW

File: prow/cmd/deck/main.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var (
 	tideURL               = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
 	hookURL               = flag.String("hook-url", "", "Path to hook plugin help endpoint.")
 	oauthUrl              = flag.String("oauth-url", "", "Path to deck user dashboard endpoint.")
-	githubOAuthConfigFile = flag.String("github-oauth-config-file", "/etc/github/app", "Path to the file containing the Git App Client secret.")
+	githubOAuthConfigFile = flag.String("github-oauth-config-file", "/etc/github/app", "Path to the file containing the GitHub App Client secret.")
```
Comment (qhuynh96) [inline prow/cmd/deck/main.go]: /etc/github/oauth-config

Comment (BenTheElder) [reply prow/cmd/deck/main.go]: done

Comment (BenTheElder) [reply prow/cmd/deck/main.go]: nope these won't work

**Comment by BenTheElder** [comment]
nope your secret looks like:
```
$ kubectl get secret github-oauth-config -o yaml
apiVersion: v1
data:
  client_id: <redacted>
  client_secret: <redacted>
  final_redirect_url: <redacted>
  redirect_url: <redacted>
...
```
Each of these keys under data will be individual files...

**Comment by BenTheElder** [comment]
/lgtm cancel
I fixed the cookie secret loading, but the github oath config needs fixing

**Comment by BenTheElder** [comment]
I've recreated the oauth secret from a file

**Comment by BenTheElder** [comment]
OK secrets all fixed, prow is back up :+1:

**Pull request merged**